notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>Looks like a cut-and-paste error happened here?</p><p> </p><p><strong>Note</strong>: Jupyter runs lines prefixed with&nbsp;<code>!</code>&nbsp;as shell commands, and it interpolates Python variables prefixed with `###&nbsp;Set up your GCP project </p><p><strong>The following steps are required, regardless of your notebook environment.</strong> </p><ol><li>into these commands.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
My cell is rendering correctly?